### PR TITLE
Attempt 1 - Manipulating Images

### PR DESCRIPTION
**Pull request by larawillson**

### CODE REVIEW

**Comment by larawillson** [comment]
@halfaro1 Hi Prof! Could you take a look at this? I'm struggling a bit - the program is having a hard time locating the images...

File: resizeAndAddLogo.py [hunk 1]
```diff
@@ -7,8 +7,42 @@
...
+   # Loop over all files in the working directory.
+for filename in os.listdir('originals'):
+
+    # skip non-image files and the logo file itself 
+
+    if not (filename.endswith('.png') or filename.endswith('.jpg')) \
+        or filename == LOGO_FILENAME:
+        continue
+    
+im = Image.open(filename)
```
Comment (halfaro1) [inline resizeAndAddLogo.py]: @larawillson why is this not indented with the rest of your `if` statements and `for` loop? If it's not indented, Python reads it as being outside of those blocks

**Comment by larawillson** [comment]
@halfaro1 I did go ahead and fix the indents but still can't get this to work with me. I was able to make the program work if the photos were in the working directory, however, I just can't get the program to look into the originals folder. :(

File: resizeAndAddLogo.py [hunk 2]
```diff
@@ -7,8 +7,28 @@
...
 logoWidth, logoHeight = logoIm.size
 
-#TODO: Loop over all files in the working directory
-#TODO: Check if file image needs to be
-#TODO: Calculate the new width and height to resize to.
-#TODO: Add the logo.
-#TODO: Save changes.
+os.makedirs('withLogo', exist_ok=True)
+# Loop over all files in the working directory.
+for filename in os.listdir(os.path.join('.','originals')):
```
Comment (halfaro1) [inline resizeAndAddLogo.py]: @larawillson if you're just going one directory deep, you don't have to make it this complicated, you can do this: `for filename in os.listdir('originals'):`

**Comment by halfaro1** [comment]
@larawillson 
Thanks for continuing to work on this project! Unfortunately, I have to grade this project as-is. I am closing out this pull request and entering your grade on Blackboard. Please see your grade there.

**Pull request closed without merging**